src/docket/dependencies.py

```diff
@@ -222,6 +224,17 @@ async def __aenter__(self) -> "Retry":
         retry.attempt = execution.attempt
         return retry
 
+    def at(self, when: datetime) -> NoReturn:
+        now = datetime.now()
```

Comment:
Oh this will need to be `datetime.now(timezone.utc)` so we aren't mixing naive with aware